### PR TITLE
use f27 base image

### PR DESCRIPTION
**Pull request by cverna**
Signed-off-by: Clement Verna <cverna@tutanota.com>

### CODE REVIEW

File: base/Dockerfile.fedora [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM fedora:26
+FROM registry.fedoraproject.org/fedora:27
```
Comment (lcarva) [inline base/Dockerfile.fedora]: This shouldn't be needed as this the [default configured](https://github.com/lcarva/osbs-box/blob/c64ad3d2e651213653b5fbc72df15565eb8674b3/shared-data/opt/local/osbs/osbs.conf#L23) `source_registry_uri`

File: hub/Dockerfile.fedora [hunk 2]
```diff
@@ -10,10 +10,11 @@ RUN dnf -y --refresh install \
         koji-hub-plugins \
         koji-utils \
         koji-web \
-        mod_auth_kerb \
+        mod_auth_gssapi \
         mod_ssl \
         mod_wsgi \
         python-requests \
+	python2-koji \
```
[the remark this answers is not in the document]
Comment (cverna) [reply hub/Dockerfile.fedora]: looks like a bug from the github diff, see the the file here --> https://github.com/cverna/osbs-box/blob/748acb86b6567b9dc9a5bf545d38be3a6b12b184/hub/Dockerfile.fedora

File: base/Dockerfile.fedora (hunk 1 above)
Comment (cverna) [reply base/Dockerfile.fedora]: This is the base image used to build the other services, if I do a docker pull fedora it will default to docker.io.

Comment (lcarva) [reply base/Dockerfile.fedora]: You are, of course, correct :)

File: hub/Dockerfile.fedora (hunk 2 above)
[the remark this answers is not in the document]
Comment (lcarva) [reply hub/Dockerfile.fedora]: Ah! You have a tab in there, can you switch to spaces instead?

File: hub/Dockerfile.fedora [hunk 3]
```diff
@@ -10,10 +10,11 @@ RUN dnf -y --refresh install \
         koji-hub-plugins \
         koji-utils \
         koji-web \
-        mod_auth_kerb \
+        mod_auth_gssapi \
         mod_ssl \
         mod_wsgi \
         python-requests \
+        python2-koji \
```
Comment (cverna) [inline hub/Dockerfile.fedora]: \o/ sneaky tabs

**Pull request merged**